.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,59 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Check Coding Standards"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - "master"
+
+jobs:
+    coding-standards:
+        name: "Check Coding Standards"
+
+        runs-on: ${{ matrix.operating-system }}
+
+        strategy:
+            matrix:
+                dependencies:
+                    - "locked"
+                php-version:
+                    - "7.4"
```

Comment:
Can we include all currently-supported versions of PHP?

Reply:
See https://github.com/thephpleague/skeleton/pull/152/commits/bab279c090fe08e01903ce296d8065e555109df2